sharpy/linear/assembler/linearbeam.py

```diff
@@ -147,30 +155,35 @@ def initialise(self, data, custom_settings=None):
         settings.to_custom_types(self.settings, self.settings_types, self.settings_default,
                                  self.settings_options, no_ctype=True)
 
-        self.settings['rigid_modes_cg'] = data.settings['Modal']['rigid_modes_cg']  # use the same value as in Modal solver
+        self.settings['rigid_modes_ppal_axes'] = data.settings['Modal']['rigid_modes_ppal_axes']  # use the same value as in Modal solver
         beam = lingebm.FlexDynamic(data.linear.tsstruct0, data.structure, self.settings)
         self.sys = beam
         self.tsstruct0 = data.linear.tsstruct0
 
-        # State variables
+        # State variables - for the purposes of dof removal PRIOR to first order system assembly
         num_dof_flex = self.sys.structure.num_dof.value
         num_dof_rig = self.sys.Mstr.shape[0] - num_dof_flex
-        state_db = {'eta': [0, num_dof_flex],
-                  'V_bar': [num_dof_flex, num_dof_flex + 3],
-                  'W_bar': [num_dof_flex + 3, num_dof_flex + 6],
-                  'orient_bar': [num_dof_flex + 6, num_dof_flex + num_dof_rig],
-                  'dot_eta': [num_dof_flex + num_dof_rig, 2 * num_dof_flex + num_dof_rig],
-                  'V': [2 * num_dof_flex + num_dof_rig, 2 * num_dof_flex + num_dof_rig + 3],
-                  'W': [2 * num_dof_flex + num_dof_rig + 3, 2 * num_dof_flex + num_dof_rig + 6],
-                  'orient': [2 * num_dof_flex + num_dof_rig + 6, 2 * num_dof_flex + 2 * num_dof_rig]}
-        self.state_variables = LinearVector(state_db, self.sys_id)
+
+        if num_dof_rig == 0:
+            state_variable_list = [
+                VectorVariable('eta', size=num_dof_flex, index=0),
+            ]
+        else:
+            state_variable_list = [
+                VectorVariable('eta', size=num_dof_flex, index=0),
+                VectorVariable('V', size=3, index=1),
```

Comment:
Comment/explain these variables, especially V and W which are not common in the beam formulation